README.adoc

```diff
@@ -211,21 +208,24 @@ interface. For this guide, you can access a bare-bones `Tracer` implementation f
 the Zipkin server in the form of a user feature for Open Liberty.
 
 This feature is already configured for you in your [hotspot file=1]`pom.xml` and [hotspot file=0]`server.xml` files. 
-It is automatically downloaded and installed into each service when you run a Maven build. 
+It's automatically downloaded and installed into each service when you run a Maven build. 
 You can find the [hotspot=zipkinUsr file=0]`opentracingZipkin` feature enabled in your [hotspot file=0]`server.xml` file.
 
-The [hotspot=download file=1]`download-maven-plugin` Maven plug-in in your `pom.xml` is responsible for downloading and installing the feature.
+The [hotspot=download file=1]`download-maven-plugin` Maven plug-in in your `pom.xml`
+downloads and installs the [hotspot=zipkinUsr file=0]`opentracingZipkin` feature.
 
 If you want to install this feature yourself, see
-https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/twlp_dist_tracing.html[Enabling distributed tracing^]
+https://www.ibm.com/docs/en/was-liberty/base?topic=environment-enabling-distributed-tracing[Enabling distributed tracing^]
```

Comment:
suggested revision for IBM Docs (Knowledge Center is no more)

If you want to install this feature yourself, see https://www.ibm.com/docs/en/was-liberty/base?topic=environment-enabling-distributed-tracing[Enabling distributed tracing^] in  IBM Documentation.